docs/content/releases/posts/v0.43.0.md

```diff
@@ -1,12 +1,36 @@
 ---
 slug: v0.43.0
-date: 2025-06-12
+date: 2025-08-13
 links:
   - v0.43.0 GitHub release: https://github.com/galasa-dev/galasa/releases/tag/v0.43.0
 ---
 
 # 0.43.0 - Release Highlights
 
+## Changes affecting the Galasa Service
+
+- Requests made to the Galasa API Server have been made more efficient to avoid errors when attempting to send a message through the gRPC protocol that exceeds the maximum size. See [#2312](https://github.com/galasa-dev/projectmanagement/issues/2312).
+
+- The Galasa ecosystem Helm chart creates the `_users` database in CouchDB if it doesn't already exist. This is a required database for CouchDB that should be created as default to prevent errors when the Result Archive Store is started.
+
+- The Engine Controller waits for the ETCD and RAS Pods to be ready in the Galasa service before completing any actions that involve the CPS/DSS/RAS.
```

Comment:
"Internal change, no externals effecting users. A deployed Galasa service will start up more cleanly."

Perhaps we need to add 'internal changes not effecting users' or something ?

Reply:
Thanks for the suggestion, this has been addressed in https://github.com/galasa-dev/galasa/pull/386/commits/450ef7cbee3774f9b6418c58b5426cc02d6bbeef